docs/implementation/infisical/infisical-patterns.md

```
@@ -0,0 +1,224 @@
+        name: appuser
+        password: "{{ db_password }}"
+        db: appdb
+        state: present
+```
+
+### Pattern 3: Use Universal Auth explicitly in playbooks
+
+If environment variables are not preferred, pass Universal Auth parameters directly to the lookup.[4][1]
+
+```yaml
+- name: Universal Auth inline usage
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)